experimental/dds/tree2/src/core/schema-stored/schema.ts

```diff
@@ -136,46 +118,166 @@ export const storedEmptyFieldSchema: TreeFieldStoredSchema = {
 	types: new Set(),
 };
 
+/**
+ * Opaque type erased handle to the encoded representation of the contents of a stored schema.
+ * @alpha
+ */
+export interface ErasedTreeNodeSchemaDataFormat extends Erased<"TreeNodeSchemaDataFormat"> {}
+export interface BrandedTreeNodeSchemaDataFormat
+	extends Brand<TreeNodeSchemaDataFormat, ErasedTreeNodeSchemaDataFormat> {}
+
+/**
+ * @alpha
```

Comment:
How does api-extractor react if you mark this internal?

Reply:
It would error. For some reason SharedTree is internal ( I think its related to a workaround for https://dev.azure.com/fluidframework/internal/_workitems/edit/6458 ), but literally nothing else in the package is, so making this @internal will just error since its used by things that are alpha. If it wasn't, it would not be exported at all. If you search the api report, you can see the three places in the alpha API that use it.